graphql-server/src/utils/servers.ts

```diff
@@ -25,17 +27,27 @@ export function initializeHttpsServer(app: Express) {
   return httpsServer;
 }
 
+export interface ApolloServerContext {
+  req: {
+    user: any;
+    isAuthenticated: boolean;
+  };
+  oncotreeCache: NodeCache;
+}
+
 export async function initializeApolloServer(
   httpsServer: https.Server,
   app: Express
 ) {
   const neo4jDbSchema = await buildNeo4jDbSchema();
-
   const mergedSchema = mergeSchemas({
     schemas: [neo4jDbSchema, oracleDbSchema],
   });
 
-  const apolloServer = new ApolloServer({
+  const oncotreeCache = new NodeCache({ stdTTL: 86400 }); // 1 day
+  await fetchAndCacheOncotreeData(oncotreeCache);
```

Comment:
This is the Initial fetch and caching of Oncotree data when the GraphQL server boots up.